code/datums/mind.dm

```diff
@@ -238,7 +238,7 @@
 
 	enslaved_to = creator
 
-	current.faction = creator.faction
+	current.faction = creator.faction.Copy()
 

```

Comment:
I get your point but this is a really dangerous way to do that, because if THIS mob loses a faction the creator has, the creator loses it TOO.